src/hatti/shared.cljs

```diff
@@ -35,7 +35,9 @@
   "An initial, empty, app-state, which can be modified to change dataviews."
   []
   (atom
-   {:views {:all [:overview :map :table :chart :saved-charts :settings]
+   {:views {:all [:overview :map :table :chart :saved-charts :settings
+                  :map-table
```

Comment:
if we don't add these to that atom can we merge this into master without changing the default functionality?